bliss/surveys/survey.py

```diff
@@ -0,0 +1,82 @@
+import pytorch_lightning as pl
+from torch.utils.data import Dataset
+
+
+class Survey(pl.LightningDataModule, Dataset):
+    def __init__(
+        self,
+        predict_device=None,
+        predict_crop=None,
+    ):
+        super().__init__()
+
+        self.predict_device = predict_device
+        self.predict_crop = predict_crop
```

Comment:
You don't need to do this if you think it's out of scope, but we should probably move these out of the survey and into prediction-specific code